python/interpret_community/lime/lime_explainer.py

```diff
@@ -194,6 +195,8 @@ def __init__(self, model, initialization_examples, is_function=False, explain_su
             outputs a 1 dimensional array.
         :type model_task: str
         """
+        if lime_import_failed:
+            raise Exception("Could not import LIME, required for LIMEExplainer")
         self._datamapper = None
```

Comment:
I that ok, if the execution now fails? Or would it have filed anyways previously as well?

Reply:
it would have failed as well since lime is not installed